content/docs/technical_practices/how_to_use_hub23.md

```diff
@@ -0,0 +1,133 @@
+1. Go to [binder.hub23.turing.ac.uk](http://binder.hub23.turing.ac.uk) and login with your GitHub username and password!
+   You will be asked to authorise an OAuth app from `binderhub-test-org`.
+   This has read-only permissions to read your organisation/team memberships and is why your membership needs to be public!
+
+## How to use the Hub23 Binder
+
+1. Copy the URL of a Binder-ready repo into the URL box on the Hub23 Binder homepage.
+   There is a drop-down menu to say whether the repo is from GitHub, GitLab, etc.
+   - You can specify a specific branch, tag or commit to build in the "Git branch, tag or commit" box.
+   - You can specify a file to be opened on launch by filling in the "Path to notebook file" box with a path relative the the repo root.
+   - You can specify an environment to be launched by changing the drop-down menu next to the "Path to notebook file" box to URL.
+     Enter `lab` for a JupyterLab environment, or `rstudio` for RStudio.
+1. Press launch!
+   The repo will be spun up on a server in the Cloud and connected to your browser.
+
+### How do you know if a repo is Binder-ready?
+
+A Binder-ready repo should contain a configuration file in its root that lists all the software needed to run the code.
+
+For Python, this may be a `requirements.txt` or `environment.yml` file.
+For R, this may be `NAME` and `DESCRIPTION` files or an `install.R` file.
+
+It must also contain a `runtime.txt` file which contains the version of the core programming language.
+For example:
+
+```bash
+python-3.6          # To pin to v3.6 of python
+r-<YYYY>-<MM>-<DD>  # To pin to a specific version of R published on MRAN
+```
+
+Just because these files exist, doesn't necessarily mean that Binder will be able to launch them first time!
+Some debugging may be required!
+
+## How to make your own repo Binder ready
+
+To make your repo Binder-ready, you need to provide a configuration file which describes your software into the root of the repo.
+
+The Binder team have provided a list of minimum working examples for a variety of configuration files [here](https://mybinder.readthedocs.io/en/latest/config_files.html#configuration-files).
+The [binder-examples](https://github.com/binder-examples) organisation is also a good place to find examples of different configurations.
+
+A very high-level, Python-based tutorial is available [here](https://bit.ly/zero-to-binder-solo) or you can read the Binder section of _The Turing Way_ book [here](https://the-turing-way.netlify.com/reproducible_environments/reproducible_environments.html#binder-1).
+
+To auto-generate config files for a Python environment:
+
+```bash
+pip freeze > requirements.txt
+conda env export > environment.yml
+```
+
+## Troubleshooting
+
+If you have any problems, check out the [mybinder documentation](https://mybinder.readthedocs.io/en/latest/), [open an issue](https://github.com/alan-turing-institute/hub23-deploy/issues/new) on the deployment repo, or ask Sarah Gibson.
```

Comment:
Sarah hasn't worked with us for a while.

In general, I think we decided to avoid using names and instead link to the _internal_ list of responsibilities/contacts in the REGistry.